webapp/src/main/resources/public/js/sdk/entity/User.js

```diff
@@ -28,7 +28,7 @@ export default class User {
      * Get best display name for user
      */
     getDisplayName() {
-        let name = this.username;
+        let name = this.createdByUserName;
```

Comment:
that should stay `username`. Was there any issue with it?

(`createdByUserName` is the user that created the user, name is improper also I think it is an id)

Reply:
you are right. It should be username instead of createdByUserName. I did not find the issue since we not show user info for branch result in dashboard.